server/permalinks.go

```diff
@@ -98,6 +98,20 @@ func (p *Plugin) processReplacement(r replacement, glClient *gitlab.Client, wg *
 		Ref: &r.permalinkData.commit,
 	}
 	projectPath := fmt.Sprintf("%s/%s", r.permalinkData.user, r.permalinkData.repo)
+
+	// Check if project is public
+	if p.getConfiguration().EnableCodePreview != "privateAndPublic" {
+		repo, _, err := glClient.Projects.GetProject(projectPath, &gitlab.GetProjectOptions{})
+		if err != nil {
+			p.API.LogError("Error while fetching project information", "error", err.Error())
```

Comment:
```suggestion
			p.API.LogError("Error while fetching project information", "Error", err.Error())
```

Reply:
ignore, if other places are using `error`

Reply:
yes, we are